stock_back2back_order_proc/__openerp__.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2015 Elico Corp (<http://www.elico-corp.com>)
+#    Author: Yannick Gouin <yannick.gouin@elico-corp.com>
+#            Alex Duan <alex.duan@elico-corp.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Stock Back-to-back orders',
+    'version': '1.0',
+    'category': 'Warehouse',
+    'description': """
+This module aims to change the original back-order logic of Odoo
+in chained locations introducing true back-to-back orders.
+
+
+STANDARD ODOO BACK-TO-BACK ORDER BEHAVIOR:
+Original behavior is not fully suitable to handle back-to-back backorders
+    process (check "back-to-back orders comparison.pdf"):
```

Comment:
Thanks for your advise,Alexandre! Sorry for this late reply since it was during Chinese New Year holiday. I'll start working on this today.:)